src/www/ui_components/server-card.html

```diff
@@ -191,7 +200,7 @@
         },
         connectButtonDisabled: {
           type: Boolean,
-          computed: "_computeConnectButtonDisabled(state)",
+          computed: "_computeConnectButtonDisabled(state, usesUnsupportedCipher)",
```

Comment:
I think this is not needed because the state will be DISCONNECTED anyway.

Reply:
I added the disabled property to address this